modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/index/ExperimentLabelIndexAccessor.java

```diff
@@ -57,6 +58,22 @@
     ResultSet updateBy(UUID uuid, Date modified, Date startTime, Date endTime, String state, String appName, String label);
 
     /**
+     * Insert experiment label index statement
+     *
+     * @param uuid
+     * @param modified
+     * @param startTime
+     * @param endTime
+     * @param state
+     * @param appName
+     * @param label
+     * @return Statement
+     */
+    @Query("update experiment_label_index set id = ?, modified = ?, start_time = ?, end_time = ?, state = ? " +
+            "where app_name = ? and label = ?")
+    Statement insertBy(UUID uuid, Date modified, Date startTime, Date endTime, String state, String appName, String label);
```

Comment:
@abhishekjain88 agree, renamed these methods to insertOrUpdateBy() & insertOrUpdateStatementBy()...